techmatters/flex-plugins
# feat: Client Profiles permissions [CHI-2550]

## Description
This PR replicates the changes of https://github.com/techmatters/hrm/pull/561 to provide a better UX. The configuration for NZ is bundled here as well.

### Checklist
- [x] [Corresponding issue has been opened](https://tech-matters.atlassian.net/browse/CHI-2550)
- [n/a] Feature flags added
- [n/a] Strings are localized
- [n/a] Tested for chat contacts
- [n/a] Tested for call contacts

### Verification steps
- Run local server `npm run dev`
- Confirm that if
  - you can't "view profile", you can't open clients from the clients list.
  - you can't "flag profile", you can't add new flags to the profile via the flags dropdown.
  - you can't "unflag profile", you can't remove flags from the profile via the flags dropdown.
  - you can't "view profile section", the entire section of that type is stripped from the UI.
  - you can't "create profile section", on a profile where there is no section of the given type, you don't have a way to create it (missing edit button).
  - you can't "edit profile section", on a profile where already exists a section of the given type, you don't have a way to edit it (missing edit button).
